gazebo_plugins/src/gazebo_ros_wheel_slip.cpp

```diff
@@ -256,6 +272,102 @@ void GazeboRosWheelSlip::Load(gazebo::physics::ModelPtr _model, sdf::ElementPtr
       }
     }
   }
+
+  double zero_wheel_spin_tolerance = 1.0e-3;
+  if (!_sdf->HasElement("zero_wheel_spin_tolerance")) {
+    RCLCPP_INFO(
+      impl_->ros_node_->get_logger(), "Missing <zero_wheel_spin_tolerance>, defaults to %f",
+      zero_wheel_spin_tolerance);
+  } else {
+    zero_wheel_spin_tolerance = _sdf->GetElement("zero_wheel_spin_tolerance")->Get<double>();
+  }
+
+  double update_rate = 100.0;
+  if (!_sdf->HasElement("update_rate")) {
```

Comment:
if this only applies to the publisher, can you rename it as `publisher_update_rate`? The underlying `WheelSlipPlugin` itself still runs with every physics step

Reply:
Good call, fixed in https://github.com/ros-simulation/gazebo_ros_pkgs/pull/1331/commits/53c792ba286876e9e477153ee02cdb5f8fa825e6.